frontend/src/features/directory/components/VendorCard.tsx

```diff
@@ -176,12 +176,22 @@ export const VendorCard = ({
                 return (
                   <Chip
                     key={index}
-                    label={specialty}
+                    label={`${specialty}`}
                     variant="outlined"
                     sx={{ fontSize: '0.875rem', fontWeight: 'medium', mt: 2 }}
                   />
                 );
               })}
+              {vendor.tags.length > 0 && (vendor.tags
+                .filter((tag) => tag.is_visible)
+                .map((tag) =>
+                  <Chip
+                    key={tag.id}
+                    label={`${tag.display_name}`}
+                    variant="outlined"
+                    sx={{ fontSize: '0.875rem', fontWeight: 'medium', mt: 2 }}
```

Comment:
nit: since we're reusing the chip styling, we should probably define a chip class. can be a follow up